imports/api/blockchain/modules/web3Util.js

```diff
@@ -0,0 +1,33 @@
+import Web3 from 'web3';
```

Comment:
Looking for feedback if /imports/api/blockchain/modules/ is a good place for this sort of new file

Reply:
Looks good for me, as long as it properly works.